peps/pep-0771.rst

```diff
@@ -0,0 +1,502 @@
+point of view, however it appears to be the best compromise as it is the only
+solution to a very real problem that would not break backward-compatibility of
+packaging infrastructure in any significant way. It is an opt-in solution which
+means that package maintainers can choose whether or not to use it, and it is
+flexible enough to accommodate both of the major use cases described in
+`Motivation`_.
+
+Specification
+=============
+
+``Default-Extra`` Metadata Field
+---------------------------------
+
+A new metadata field, ``Default-Extra``, will be added to the `core package
+metadata <https://packaging.python.org/en/latest/specifications/core-metadata/#core-metadata>`_.
+This field allows package maintainers to define an extra that is
+automatically included when a user installs the package without specifying any
+extras explicitly::
+
+    Default-Extra: recommended
+
+If multiple default extras are needed, one ``Default-Extra:`` entry
+should be provided for each one::
+
+    Default-Extra: backend1
+    Default-Extra: backend2
+    Default-Extra: backend3
+
+New key in ``[project]`` metadata table
+---------------------------------------
+
+A new key will be added to the ``[project]`` table in project metadata as
+originally defined in :pep:`621` and now defined in the `PyPA specifications
+<https://packaging.python.org/en/latest/specifications/>`_. This key will be named
+``default-optional-dependencies`` with the following description:
+
+* `TOML <https://toml.io/>`_ type: Array of strings
+* Corresponding core metadata field: ``Default-Extra``
+
+Each string in ``default-optional-dependencies`` should be the name of an extra
+defined in `optional-dependencies
+<https://packaging.python.org/en/latest/specifications/pyproject-toml/#dependencies-optional-dependencies>`_,
+and each extra in this array will be converted to a matching ``Default-Extra``
+entry in the core package metadata. Examples of valid usage which would
+produce the example ``Default-Extra`` entries presented in the previous section are:
+
+.. code-block:: toml
+
+    [project]
+    default-optional-dependencies = [
+        "recommended"
+    ]
+
+and:
+
+.. code-block:: toml
+
+    [project]
+    default-optional-dependencies = [
+        "backend1",
+        "backend2",
+        "backend3"
+    ]
+
+
+Overriding default extras
+-------------------------
+
+If extras are explicitly given in a dependency specification, the default
+extras are not installed. Otherwise, the default extras are used.
+
+For example, if a package
+defines a ``extra1`` default extra as well as a non-default ``extra2``
+extra, then if a user was to install the package with::
+
+    pip install package
+
+the ``extra1`` dependency would be included. If the user instead uses::
+
+    pip install package[extra2]
+
+then the ``extra1`` extras would not be installed.
+
+If the same package is specified multiple times in a dependency tree, then if
```

Comment:
I find this paragraph difficult to parse and specifying.  Can you rephrase for clarity?